core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-editor-modal.controller.ts

```diff
@@ -128,6 +128,7 @@ angular.module('oppia').controller('QuestionSuggestionEditorModalController', [
         if (AlertsService.warnings.length === 0) {
           $scope.skillDifficulty = result.skillDifficulty;
           $scope.setDifficultyString($scope.skillDifficulty);
+          $scope.$digest();
```

Comment:
Probably better to use `$scope.$applyAsync()`, we have used it in several places in the codebase.

Reply:
Done! PTAL